ctdcal/oxy_fitting.py

```diff
@@ -658,6 +643,8 @@ def calculate_weights(pressure):
 
     return weights
 
+"""code_pruning: no calls, seems related to PMEL oxygen code?
```

Comment:
The code doesn't really even do what it's defined as. This is extending the bounds on either end to prevent data loss during interpolation, which is the next step (not in these functions).
I have already adopted PMEL interpolation/methodology in [oxy_fitting.match_sigmas()](https://github.com/cchdo/ctdcal/blob/master/ctdcal/oxy_fitting.py#L781-L815), which is why I feel confident in deleting these.

Reply:
Fair enough, just cut them all.